v2/apiserver/schemas/role-assignment.json

```diff
@@ -50,7 +50,7 @@
 				"scope": {
 					"type": "string",
 					"description": "The event source this role should be scoped to",
-					"pattern": "^[a-z][a-z\\d-]*[a-z\\d]$",
+					"pattern": "^[a-z][a-z\\d./-]*[a-z\\d]$",
```

Comment:
This still won't allow a lot of characters that are valid in URLs like _ and ?, but I think that's ok. This is a pretty reasonable regex for the purpose.

I do think we should allow uppercase alphas though.